src/skydb.ts

```diff
@@ -86,7 +101,26 @@ export async function getJSON(
   if (entry === null) {
     return { data: null, dataLink: null };
   }
-  const dataLink = entry.data;
+
+  // Determine the data link.
+  // TODO: Can this still be an entry link which hasn't yet resolved to a data link?
+  let dataLink = entry.data;
+  if (typeof dataLink === "string") {
```

Comment:
With last changes, I don't think we should ever see a string here.

Reply:
Egads!